cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -1029,10 +1109,29 @@ def setUp(self):
                 {u'id': 1, u'name': u'Group B', u'version': 1, u'usage': []},
             ],
         }
+        experiment_groups_list = {
```

Comment:
It would be good for the test case to have a couple of experiment partitions as well as a content group partition, just to ensure that the logic works in more than just the simple case.